generators/entity/templates/client/angular/src/main/webapp/app/entities/_entity-management-dialog.component.html

```diff
@@ -62,7 +62,7 @@ <h4 class="modal-title" id="my<%= entityClass %>Label" jhiTranslate="<%= keyPref
                 const values = fields[idx].fieldValues.replace(/\s/g, '').split(',');
                 for (key in values) {
                     const value = values[key]; _%>
-                <option value="<%= value %>" jhiTranslate="<%=enumPrefix%>.<%=value%>"><%= value %></option>
+                <option value="<%= value %>">{{'<%=enumPrefix%>.<%=value%>' | translate}}</option>
```

Comment:
you need to handle case when translation is disabled as well. For that `<%= value %>` alone is required